test/powershell/Modules/Microsoft.PowerShell.Utility/Implicit.Remoting.Tests.ps1

```diff
@@ -2076,3 +2076,42 @@ Describe "GetCommand locally and remotely" -tags "Feature" {
         $localCommandCount | Should Be $remoteCommandCount
     }
 }
+
+Describe "Import-PSSession on Restricted Session" -tags "Feature","RequireAdminOnWindows","Slow" {
+
+    BeforeAll {
+
+        # Skip tests for non Windows
+        if (! $IsWindows)
+        {
+            $originalDefaultParameters = $PSDefaultParameterValues.Clone()
+            $PSDefaultParameterValues["it:skip"] = $true
+        }
+        else
+        {
+            New-PSSessionConfigurationFile -Path $TestDrive\restricted.pssc -SessionType RestrictedRemoteServer
+            Register-PSSessionConfiguration -Path $TestDrive\restricted.pssc -Name restricted -Force
+            $session = New-PSSession -ComputerName localhost -ConfigurationName restricted
+        }
+    }
+
+    AfterAll {
+
+        if ($originalDefaultParameters -ne $null)
+        {
+            $PSDefaultParameterValues = $originalDefaultParameters
```

Comment:
Should be `$global:PSDefaultParameterValues = $originalDefaultParameters`. Otherwise, you are just defining a variable in the local scope.